tests/cucumber/steps/steps.js

```diff
@@ -41,6 +40,10 @@ async function loadResource(res) {
   });
 }
 
+async function loadResourceAsJson(res) {
+  return JSON.parse(await loadResource(res));
```

Comment:
`loadResource` returns a `Buffer`, I think you'll need to convert to a string before JSON parsing

Reply:
Interesting, I didn't see this originally converted to a string, but I added the conversion just in case [ee3e2e1](https://github.com/algorand/js-algorand-sdk/pull/801/commits/ee3e2e12b1260fad5a966352d16dfb256f0906e2)